lib/galaxy/config.py

```diff
@@ -184,7 +184,9 @@ def __init__( self, **kwargs ):
         self.admin_users_list = [u.strip() for u in self.admin_users.split(',') if u]
         self.mailing_join_addr = kwargs.get('mailing_join_addr', 'galaxy-announce-join@bx.psu.edu')
         self.error_email_to = kwargs.get( 'error_email_to', None )
-        self.activation_email = kwargs.get( 'activation_email', None )
+        # activation_email was used until release_15.03
+        activation_email = kwargs.get( 'activation_email', None )
+        self.email_from = kwargs.get( 'email_from', activation_email )
```

Comment:
Is there any way to go ahead and figure out hostname (for use in <galaxy-no-reply@HOSTNAME>) here?

If not, this looks great.  If so, we should do that.